docs/contribute/definitions.md

```diff
@@ -6,8 +6,13 @@ html_theme.sidebar_secondary.remove: true
 
 In order to speak with the same vocabulary, here is some words that have a specific meaning in this project:
 
-- **Module**: a group of features that share a common purpose and that are contained in a namespace (i.e. datapack structures)
-- **Feature**: something designed for the final user in order to allow him to perform a task. Most of the time, a feature is a function tag to call instead of the function. This enable, among other things, filtering the `/function` completion by proposing only the features. A feature can also be a loot_table, a structure, a predicate or whatever.
-- **Brigadier**: This is the command completer and highlighter of Minecraft that you can see when you enter a command in the chat. We define it here because most of people doesn't know that this system have a name.
-- **Data pack developer**: A person who develops a datapack (using Bookshelf or not).
-- **Bookshelf contributor**: A person contributing to the development of Bookshelf, through pull request for instance.
+---
+
+:::{list-table}
+*   - **Module**
+    - A group of features that share a common purpose and that are contained in a namespace (i.e. datapack structures).
+*   - **Feature**
+    - Something designed for the final user in order to allow him to perform a task. Most of the time, a feature is a function tag to call instead of the function. This enable, among other things, filtering the `/function` completion by proposing only the features. A feature can also be a loot_table, a structure, a predicate or whatever.
+*   - **Brigadier**
+    - This is the command completer and highlighter of Minecraft that you can see when you enter a command in the chat. We define it here because most of people doesn't know that this system have a name.
```

Comment:
"completer" is not an existing English word.

Reply:
haha this one must be old